app/src/main/java/fr/free/nrw/commons/di/CommonsApplicationModule.java

```diff
@@ -94,6 +99,17 @@ public ContentProviderClient provideCategoryContentProviderClient(Context contex
         return context.getContentResolver().acquireContentProviderClient(BuildConfig.CATEGORY_AUTHORITY);
     }
 
+    /**
+     * This method is used to provide instance of DepictsContentProviderClient
+     * @param context context
+     * @return DepictsContentProviderClient*/
+
+    @Provides
+    @Named("depicts")
+    public ContentProviderClient provideDepictsContentProviderClient(Context context) {
+        return context.getContentResolver().acquireContentProviderClient("fr.free.nrw.commons.beta.depicts.contentprovider");
```

Comment:
Shouldn't this be based on package name ? , getPackageName()+".depicts.contentprovider'

Reply:
Done